apiserver/server_costs_handler.go

```diff
@@ -0,0 +1,19 @@
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/alphagov/paas-billing/eventio"
+	"github.com/labstack/echo"
+)
+
+func TotalCostHandler(store eventio.TotalCostReader) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		planGUIDSByCost, err := store.GetTotalCost()
```

Comment:
`planGUIDSByCost` makes me think the return value is a `map[PlanGuid]Cost`, but it's actually a `[]TotalCost`. I think `costTotals` would be a better variable name.